src/layouts/MainLayout/assets/homeIcon.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="15.81"
+    height="17.106"
```

Comment:
Can you add these as props to all the SVGs?

- color: to control the color and fill properties if needed
- width: what if we need to make the SVGs bigger or smaller?
- height: add a default of "auto" or the same "width", but also can be a user-defined value, as in this case
- classname: just in case hehe

Reply:
> - color: to control the color and fill properties if needed
> - width: what if we need to make the SVGs bigger or smaller?
> - height: add a default of "auto" or the same "width", but also can be a user-defined value, as in this case

I propose to use for default "currentcolor" property instead of a prop "color" on the component, then it will inherit the parent text color as if the SVG was just text.

The width and height I think should be defined by the designer, and in case of want to change it, it can be changed by SCSS, in this case, I have selected the parent with CSS, for example:

```scss
   .button {
       color: gray;
       
       svg {
          width: 100%;
          height: 100%;
       }
   }
```

Another way is to add a class to the icon and change the width/height or just to pass and style the object

---

> - classname: just in case hehe

I've double checked and on VueJS just is needed to pass className if you want modify a element that is not on the root of the document, for example:

```vue
<template>
   <div class="inputContainer">
       <div class="anotherContainer">
           <input :class="inputClass">
           <img :class="iconClass" :src="icon" />
       </div>
   <div>
</template>

<script>
export default {
    name: 'InputWithIcon',
    props: {
        icon: {
           type: String,
        },
        inputClass: {
           type: [String, Array, Object]
        },
        iconClass: {
           type: [String, Array, Object]
        }
    }
}
</script>
```

In this example's case, we don't want to modify the div parent, and as the child is so nested, modifying it just using SCSS could be hard, then we need to pass some custom class props to modify these children easily (in the example they're `inputClass` and `iconClass`). But if we want to modify the root element (the top element on the component) we could just pass the class attribute normally.

Here is the doc about it: https://vuejs.org/guide/essentials/class-and-style.html#with-components